src/MassTransit/Serialization/JsonConverters/HostInfoJsonConverter.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+namespace MassTransit.Serialization.JsonConverters
+{
+    using MassTransit.Metadata;
+    using System;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    internal class HostInfoJsonConverter : JsonConverter<HostInfo>
```

Comment:
There are actually a dozen+ classes that need to be supported this way.
https://github.com/MassTransit/MassTransit/blob/develop/src/MassTransit/Serialization/JsonConverters/InternalTypeConverter.cs#L21

Reply:
why interface types like HostInfo doesn't start with I ?

Reply:
Because I don't like putting I in front of names, it's like Hungarian all over again. Stupid COM leftover, thanks Microsoft.

That said, many of the interfaces which are actual service-type interfaces, include I now. But anything that's a data container essentially (and all pipe contexts) omit it for the sake of readability.

And, again, I think the I was freaking stupid.

IHateIt.